acquia_cms.profile

```diff
@@ -46,7 +46,6 @@ function acquia_cms_form_cohesion_account_settings_form_alter(array &$form) {
     // configuration import and cohesion style rebuild functionality into
     // separate submit handlers.
     // @see \Drupal\cohesion_website_settings\Controller\WebsiteSettingsController::batch
-    $form['#submit'][] = 'acquia_cms_rebuild_cohesion';
```

Comment:
@saurabhtripathics - why is this removed?

Reply:
this code should be there, was experimenting something on local, added this code.